tokens/connect/metamask.md

```diff
@@ -38,15 +38,13 @@ Learn [how to integrate a Connect MetaMask button](/builders/interact/metamask-d
 <style>.embed-container { position: relative; padding-bottom: 56.25%; height: 0; overflow: hidden; max-width: 100%; } .embed-container iframe, .embed-container object, .embed-container embed { position: absolute; top: 0; left: 0; width: 100%; height: 100%; }</style><div class='embed-container'><iframe src='https://www.youtube.com/embed/ywpc1UwpIyg' frameborder='0' allowfullscreen></iframe></div>
```

Comment:
It seems that one of the videos is not available anymore

Reply:
this was one of the fixes included in the PR I merged last night 🙂 

I'll merge master into this PR!